src/tools/gamemodelconverter/src/chunksmanager.h

```diff
@@ -20,6 +20,11 @@ struct ChunkInfo
 
     ChunkInfo(const char *name, const StringClass &type, const StringClass &data, void *value) :
         name(name), type(type), data(data), value(value) {}
+
+    ~ChunkInfo()
+    {
+        delete [] value;
```

Comment:
This is risky. Allocation ownership should be well defined. Either use unique_ptr or a different strategy. I am surprised this compiles, because void cannot be destructed!?